src/App.scss

```diff
@@ -1 +1,26 @@
-// style go here
+html {
+  background: linear-gradient(
+    to bottom right,
+    transparent 48%,
+    #efefef 48%,
+    #efefef 52%,
+    transparent 52%
+  ),
+    linear-gradient(
+    to bottom left,
+    transparent 48%,
+    #efefef 48%, 
+    #efefef 52%,
+    transparent 52%
+  );
+  background-size: 0.25rem 0.25rem;
```

Comment:
It's really weird to set background styles to html tag, let's move them to your body, or even to the .App styles